Macroist-SwiftUI/Service/Models/MacroMeal.swift

```diff
@@ -27,10 +30,12 @@ public struct MacroMeal: Equatable, Hashable, Codable {
     public private(set) var carbs: CGFloat = .zero
     public private(set) var fat: CGFloat = .zero
     
-    public init(id: UUID = .init(),
-                mealName: String,
-                ingredients: [Ingredient] = [.init()]) {
+    public init(id: UUID,
+                dateKey: DateEntryKey = .init(),
```

Comment:
kinda feels roundabout to avoid using Timestamp directly. might need some attention later on